contrib/mk-tls-keypair.sh

```diff
@@ -50,13 +51,14 @@ openssl req -new -config ${conf} -key ${keyfile} -out ${csrfile}
 dns=$(openssl req -text -noout -verify -in $csrfile | grep DNS:)
 echo "signing with dns name: $dns" 1>&2
 cat >${tmpdir}/server.ext <<EOF
-keyUsage=critical, digitalSignature, keyEncipherment, keyAgreement
+keyUsage=critical, digitalSignature, keyEncipherment, keyAgreement, keyCertSign
 extendedKeyUsage=critical, serverAuth
 subjectAltName=$dns
+basicConstraints = critical, CA:true, pathlen:0
```

Comment:
this script is for creating a TLS cert - not a CA.

Reply:
from what I understand it's a CA (just intermediate). I'm updating it to the factory with:
```
fioctl keys ca update
```
And the factory is demanding that the `keyCertSign` is in the cert.

Reply:
there are two keys here and you are mixing them up.

1) An intermediate CA to sign requests with
2) A TLS cert for web traffic

Our backend needs to know about 1 so our device-gateway can trust devices signed with certs it produces. This script is creating 2.